mixins/js/ez-tabs.js

```diff
@@ -78,7 +78,7 @@ window.eZ = window.eZ || {};
 
                     e.preventDefault();
                     if ( this._dispatchTabChange(label, panel) ) {
-                        this._changeTab(panel, label);
+                        this._changeTab(panel, label, e);
```

Comment:
instead of passing the event object until `_unselectTab`, I think you should compute the *tab container* here to have:
```js
const tabContainer = e.target.closest('.ez-tabs');

this._changeTab(tabContainer, panel, label);
```